singularity_def/rhessys/Dockerfile_rhessysEC7.2

```diff
@@ -0,0 +1,56 @@
+FROM ubuntu:xenial
+
+# install only the packages that are needed
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    software-properties-common python-software-properties \
+    ca-certificates \
+    git \
+    make \
+    liblapack-dev \
+    vim
+
+# set environment variables for docker build
+ENV F_MASTER /code
+ENV FC gcc
+ENV FC_EXE gcc
+ENV INCLUDES -I/usr/include
+ENV LIBRARIES '-L/usr/lib -lnetcdff -llapack -lblas'
+
+RUN git clone https://github.com/NCAR/summa.git /tmp/summa \
+    && cd /tmp/summa \
+
+###################################
+## create docker user
+RUN useradd -ms /bin/bash docker
+RUN usermod -aG sudo docker
+RUN chmod -R 777 /home/docker/
+
+###################################
+# Python 3.7 (Install py37 conda to avoid mpi4py issue, see below)
+RUN wget https://repo.continuum.io/miniconda/Miniconda3-py37_4.8.3-Linux-x86_64.sh \
+    && bash Miniconda3-py37_4.8.3-Linux-x86_64.sh -b -p /opt/miniconda3 \
+    && rm Miniconda3-py37_4.8.3-Linux-x86_64.sh \
+    && chown -R docker:docker /opt/miniconda3
+    
+#Set environment variables
+ENV PATH="/opt/miniconda3/bin:${PATH}"
+
+# Install SUMMA conda env
+
+RUN /opt/miniconda3/bin/python -m pip install --quiet git+https://github.com/DavidChoi76/pyrhessys.git
```

Comment:
@DavidChoi76  is this the only lib that needs to be installed for rhessys? it has both PyRhessys and Rhessys binary?